src/mumble/Log.ui

```diff
@@ -193,6 +208,48 @@
         </property>
        </widget>
       </item>
+      <item row="1" column="1">
+       <widget class="QSpinBox" name="qsbMessageThresholdUsers">
+        <property name="enabled">
```

Comment:
This has to be initialized depending on whether the corresponding checkbox is checked or not. Currently it always starts as enabled even if the checkbox is not checked. Only after checking and un-checking the checkbox, will this get disabled.

Reply:
I could've sword I check that, and I thought the signal/slot functionality was handling it for me...